warehouse/models/mart/gtfs_quality/fct_monthly_reports_site_organization_file_checks.sql

```diff
@@ -0,0 +1,79 @@
+{{ config(materialized='table') }}
+
+WITH files_of_interest AS (
+    SELECT 'shapes.txt' AS filename, 'shapes' AS gtfs_filename, 'Visual display' AS reason
```

Comment:
nit: there are enough lines it might be worth using the [array method](https://github.com/cal-itp/data-infra/blob/main/warehouse/models/payments_views/payments_feeds.sql)

Reply:
sure, done